scripts/aws_start_app.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/bash
 
-echo "Starting application"
-sudo service notifications-api start
\ No newline at end of file
+if [ -e "/etc/init/notifications-api.conf" ]
+then
+  echo "Starting api"
+  sudo service notifications-api start
```

Comment:
I would normally use this the other way round, run the script with sudo and then if you need to run something as a specific user use that in the command eg. http://unix.stackexchange.com/questions/190571/sudo-in-non-interactive-script